pom.xml

```diff
@@ -173,9 +173,8 @@
         <hadoop.version>3.3.4</hadoop.version>
         <htrace.version>3.1.0-incubating</htrace.version>
         <httpclient.version>4.4.1</httpclient.version>
-        <jackson.spark.datatype.version>2.12.6</jackson.spark.datatype.version>
-        <jackson.spark.version>2.12.6</jackson.spark.version>
-        <jackson.version>2.12.6</jackson.version>
+        <jackson.version>2.14.1</jackson.version>
+        <jackson.databind.version>2.14.1</jackson.databind.version>
```

Comment:
Is there a point these two are separate?

Reply:
Yes, there is - `jackson-module-scala` (and some other jackson libraries, too) use standard `major.minor.patch` versioning, while the `jackson-databind` also uses micro patches `major.minor.patch.micro_patch` with some releases - mainly for vulnerabilities.

This was exactly the case for jackson `jackson-module-scala` (latest 2.12) - we needed to use `2.12.7.1` for the  `jackson-databind`, because the matching version (`2.12.7`) of the databind exists, but is vulnerable. Therefore you could not cover it with single version string.

Currently, it is ok, `2.14.1` is not known to be vulnerable, so it seems redundant, but it can easily happen that at some point a `jackson-databind` micro patch may appear and we would want to use it. Looking at the [jackson-databind version release history](https://mvnrepository.com/artifact/com.fasterxml.jackson.core/jackson-databind), it tends to happen. Therefore I find it beneficial to keep this two separate.